x-pack/plugins/security_solution/server/lib/detection_engine/signals/preview/alert_instance_factory_stub.ts

```diff
@@ -49,4 +50,7 @@ export const alertInstanceFactoryStub = <
   hasContext() {
     return false;
   },
+  getUuid() {
```

Comment:
The "preview" code somehow manages to run the rule executor locally, and so needs to have stub versions of everything available, typing wise.